diff -up pinentry-master/configure.ac.2~ pinentry-master/configure.ac
--- pinentry-master/configure.ac.2~	2024-02-25 22:39:16.473901315 +0100
+++ pinentry-master/configure.ac	2024-02-25 22:41:50.883532879 +0100
@@ -798,14 +798,14 @@ AM_CONDITIONAL(BUILD_PINENTRY_FLTK, test
 # Please change the order only after verifying that the preferred
 # pinentry really is better (more feature-complete and more secure).
 
-if test "$pinentry_gtk_2" = "yes"; then
-  PINENTRY_DEFAULT=pinentry-gtk-2
+if test "$pinentry_qt6" = "yes"; then
+  PINENTRY_DEFAULT=pinentry-qt
 else
-  if test "$pinentry_qt6" = "yes"; then
-    PINENTRY_DEFAULT=pinentry-qt
+  if test "$pinentry_qt5" = "yes"; then
+    PINENTRY_DEFAULT=pinentry-qt5
   else
-    if test "$pinentry_qt5" = "yes"; then
-      PINENTRY_DEFAULT=pinentry-qt5
+    if test "$pinentry_gtk_2" = "yes"; then
+      PINENTRY_DEFAULT=pinentry-gtk-2
     else
       if test "$pinentry_gnome_3" = "yes"; then
         PINENTRY_DEFAULT=pinentry-gnome3
